pkg/daemon/daemon.go

```diff
@@ -74,7 +73,7 @@ type Daemon struct {
 
 	nodeState *sriovnetworkv1.SriovNetworkNodeState
 
-	LoadedPlugins map[string]VendorPlugin
+	enabledPlugins map[string]plugin.VendorPlugin
```

Comment:
enabled means this are the plugins in use by the config daemon.

for example on a system with intel only it will only enabled that one or for hwoffload it will load both k8s and mlx so I think the node is fine :)

Reply:
ack